morph.go

```diff
@@ -53,6 +53,7 @@ var (
 
 	tempDir, tempDirErr  = ioutil.TempDir("", "morph-")
 	assetRoot, assetsErr = assets.Setup()
+	keepGCRoot           = *app.Flag("keep-result", "Keep latest build in .gcroots to prevent it from being garbage collected").Default("True").Bool()
```

Comment:
I'm having trouble enabling/disabling this feature? It seems like it's always disabled. I think this ends up being a pointer to a pointer.  Should probably just be `app.Flag(....` instead of `*app.Flag`.